.github/workflows/gh-container-image-evm.yml

```diff
@@ -6,9 +6,9 @@ on:
       - evm-domain
 
 env:
-  PROCESSOR_NAME: ghcr.io/${{ github.repository_owner }}/blockexplorer-evm-processor:latest
-  API_SERVER_NAME: ghcr.io/${{ github.repository_owner }}/blockexplorer-evm-api-server:latest
-  HEALTH_CHECK_NAME: ghcr.io/${{ github.repository_owner }}/health-check:latest
+  PROCESSOR_NAME: ghcr.io/${{ github.repository_owner }}/blockexplorer-evm-processor:evm-domain
```

Comment:
I think `latest` is ok, since it already has `evm` in the name

Reply:
I think is better to have evm-domain, because it can differentiate the packages deployments here, maybe name it evm-latest instead:

![image](https://github.com/subspace/blockexplorer/assets/5139220/0c1d4949-910f-47ac-911f-9301e41a6885)